freqtrade/rpc/telegram.py

```diff
@@ -206,7 +207,44 @@ def _status_table(bot: Bot, update: Update) -> None:
 
         send_msg(message, parse_mode=ParseMode.HTML)
 
+@authorized_only
+def _daily(bot: Bot, update: Update) -> None:
+    """
+    Handler for /daily <n>
+    Returns a daily profit (in BTC) over the last n days.
+    Default is 7 days
+    :param bot: telegram bot
+    :param update: message update
+    :return: None
+    """
+    trades = Trade.query.order_by(Trade.close_date).all()
+    today = date.today().toordinal()
+    profit_days = {}
+    
+    try:
+        timescale = update.message.text.replace('/daily', '').strip()
+    except:
+        timescale = 7
+
```

Comment:
If you want to support positive integers I'd suggest to make the check at this point and return like @glonlas suggested. So e.g.
```
if timescale < 0:
  send_msg('*Daily [n]:* `must be an integer greater than 0`', bot=bot)
  return
```